install.sh

```diff
@@ -49,7 +49,7 @@ deactivate
 echo "Installing plugin in $HOME/.config/GIMP/2.99/plug-ins"
 for d in openvino_utils semseg_ov stable_diffusion_ov superresolution_ov; do
     mkdir -p "$HOME/.config/GIMP/2.99/plug-ins/$d"
-    rsync -a gimpenv3/lib/python*/site-packages/gimpopenvino/plugins/"$d" "$HOME/.config/GIMP/2.99/plug-ins/."
+    rsync -a "$script_dir/gimpopenvino/plugins/$d" "$HOME/.config/GIMP/2.99/plug-ins/."
```

Comment:
The issue with this line is that it will miss the gimp_openvino_config.json file copy to the default config_dir_path in the case that an Environment variable is not used. 

The goal here is to copy from the $script_dir because of the executable permissions change needed?

Reply:
Ah yeah I missed that. Yes, the goal is to avoid running `chmod +x` on the plugin files from `complete_install.py` because the library is already installed at this point...so this causes some problems for a containerized packaging format like snaps use. It's not a major issue, just a nice-to-have.

I could just settle for wrapping the `chmod`'s with a try-except. What do you think?

Reply:
if it is not a major issue, we can bin it for now and wrap the `chmod` commands with try-except. i think the real solution is to  re-think how the config file gets created the first time.

Reply:
Ok I just reverted all the `chmod` -related changes for the plugins. I didn't even bother with a try-except because `suprocess.call()` does not throw an exception even if the shell command fails.

Reply:
Also let me know if you'd like me to clean up the commits. I often use the squash merge method from GitHub but if you prefer I clean up the commits locally just let me know. :)